src/main/java/hw/managers/taskmanagers/InMemoryTaskManager.java

```diff
@@ -30,34 +30,34 @@ private int getNextId() {
     // 2. Методы для каждого из типа задач(Задача/Эпик/Подзадача):
     // 2.a Получение списка всех задач.
     @Override
-    public ArrayList<Task> getAllTasks() {
+    public ArrayList<Task> getTasks() {
```

Comment:
Метод должен возвращать интерфейс, а не конкретную реализацию. Так код будет более гибким и не будет зависеть от конкретной реализации
```suggestion
    public List<Task> getTasks() {
```
Поправь, пожалуйста, для всех методов, которые возвращают `ArrayList`